.github/workflows/test.yml

```diff
@@ -13,11 +13,11 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [ "3.6", "3.7", "3.8", "3.9", "pypy3" ]
+        python-version: [ "3.6", "3.7", "3.8", "3.9", "pypy3.10" ]
```

Comment:
Could you please add the missing version we support from [this list](https://devguide.python.org/versions/)?